### PR TITLE
Add Babel ES6 and mocha dependencies to package.json

### PR DESCRIPTION
**Pull request by Hikari9**
Commands 'npm test' and 'npm run fix' should now work without errors with this commit.

### CODE REVIEW

**Comment by Hikari9** [comment]
My bad, I forgot that node v8 requires `package-lock.json` to be committed to the source tree now. Will fix this in the next commit.

**Review by Vaan52 — APPROVED** [review]

File: tests/sample.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import * as assert from 'assert';
+// const assert = require('assert');
```
Comment (erocrizs) [inline tests/sample.js]: please clean code

**Review by erocrizs — CHANGES_REQUESTED** [review]
minor requests

File: tests/sample.js (hunk 1 above)
Comment (Hikari9) [reply tests/sample.js]: Done. Also merged with master before this change, please review latest commit.

**Review by erocrizs — APPROVED** [review]

**Pull request merged**